src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContext.cs

```diff
@@ -37,6 +42,15 @@ public class TagHelperContext
         public IDictionary<string, object> AllAttributes { get; }
 
         /// <summary>
+        /// Gets the collection of items used to communicate with child <see cref="ITagHelper"/>s.
```

Comment:
"child" isn't quite correct; perhaps "other", "in-scope", or "parent and child"?